content/docs/user-guide/networking/multiplayer/split.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Client and Server Launchers with the Multiplayer Gem
+description: Build Open 3D Engine launchers with the Multiplayer Gem specifically targeting Clients, Servers or both.
+linkTitle: Separating Client and Server
+---
+
+The Multiplayer Gem supports builds that contain only client logic, only server logic, or both client and server logic. This allows users to create executables of smaller size by excluding unnecessary logic and dependencies. It also allows hiding potentially sensitive logic unique to one executable from the other. 
+
+The splitting functionality produces multiple build types.
+* GameLauncher is a client only launcher
+* ServerLauncher is a server only launcher suitable for Dedicated Servers
+* UnifiedLauncher provides both and is suitable for Client Hosted Servers
```

Comment:
So basically, if I ever want to switch from supporting only DedicatedServer to now supporting Client-Server, I need to stop giving players the ClientLauncher and instead give them the Unified Launcher?

Reply:
Practically speaking, that's probably how it would work out. A developer would likely want _any_ client to be able to host so they likely wouldn't use the ClientLauncher. They could still make use of the ServerLauncher for Dedicated hosting if they wanted.

Reply:
Depends on a game type, an MMO might never used Unified but a regular co-op game would.